tools/msvs/install_tools/install_tools.bat

```diff
@@ -47,7 +49,31 @@ echo script is at your own risk. Please read the Chocolatey's legal terms of use
 echo and the Boxstarter project license as well as how the community repository
 echo for Chocolatey.org is maintained.
 echo.
-echo You can close this window to stop now.
 pause
 
+cls
+echo !!!!!WARNING!!!!!
+echo -----------------
+echo Use of Boxstarter may reboot your computer automatically multiple times.
+echo When performing a reboot, Boxstarter will need to disable User Account
+echo Control (UAC) to allow the script to run immediately after the reboot. When
+echo the scripts have completed, Boxstarter will re-enable UAC. If you prematurely
+echo stop the process, UAC will need to be re-enabled manually.
+echo.
+echo Sometimes the scripts may install all necessary Windows Updates which
+echo could cause a high number of reboots that appear to be a reboot loop when
+echo in fact it is just a normal Windows Updates reboot cycle.
+echo.
+echo If this is not what you would like to occur, you can close this window
+echo to stop now.
+:acceptretry
+echo.
+echo Your computer may REBOOT SEVERAL TIMES WITHOUT FURTHER WARNING.
+echo Please type YES followed by enter to confirm that you have saved all your
+set /p "ACCEPT_PROMPT=work and closed all open programs: "
+if /i not "%ACCEPT_PROMPT%"=="yes" (
```

Comment:
Does this handle case sensitivity? You are asking for "YES", and this accepts "yes"

Reply:
The `/i` switch is for ignoring case. I used uppercase above to make it obvious that only `yes` needs to be typed, but the user can type in any case.